alb_listener/variables.tf

```diff
@@ -59,7 +59,7 @@ variable "target_deregistration_delay" {
 
 variable "target_stickiness" {
   description = "List(optional, []): An ALB target_group stickiness block"
-  type        = list(string)
+  type        = list(map(string))
```

Comment:
please also update the documentation for this change

Reply:
right, done

Reply:
You'll also need to update the variable description field. I'd remove the `List(optional, []): ` prefix from there imo, it's a bit redundant, just leave it in the README documentation